### ISSUE TITLE
PViC Fine-tune

### ISSUE THREAD

**Issue by AlineAssakaTani** [fredzzhang/pvic#50]
Hello, thank you for the amazing work with the PViC.

Could you help me? I've conducted training with PViC on my own images, and it works well, but I would like to fine-tune this model with new images, is it possible?

(Other question: there is a reason to use test partition to train on code? -> ap = self.test_hico())
Thank you!

**Comment by fredzzhang** [comment]
Hi @AlineAssakaTani,

It should be possible. You just need to build your own dataset. You can refer to [utils.py](https://github.com/fredzzhang/pvic/blob/main/utils.py) and [hicodet.py](https://github.com/fredzzhang/hicodet/blob/047bbdc0dcd7d5caf0d90634b465001d92a4df9d/hicodet.py) to find the relevant code.

The test mAP was computed just for logging purposes mostly.  If I remember this correctly, the choices hyper-parameters was determined from a validation set, which was randomly sampled from the training set. The method used to split the dataset was [here](https://github.com/fredzzhang/hicodet/blob/047bbdc0dcd7d5caf0d90634b465001d92a4df9d/hicodet.py#L268).

Fred.
